lib/retrieval-stats.js

```diff
@@ -151,8 +151,7 @@ const addHistogramToPoint = (point, values, fieldNamePrefix = '') => {
   }
 }
 
-// TODO: include Filecoin Deal id
-export const getTaskId = (/** @type {import('./typings').Measurement} */m) => `${m.cid}`
+export const getTaskId = (/** @type {import('./typings').Measurement} */m) => `${m.cid}::${m.minerId}`
```

Comment:
FYI: I first updated a test to fail with an expected error and only then made this change.